Assets/MixedRealityToolkit/Inspectors/Profiles/DataProviderAccessServiceInspector.cs

```diff
@@ -196,6 +196,17 @@ protected bool RenderDataProviderEntry(int index, GUIContent removeContent, Syst
 
                     serializedObject.ApplyModifiedProperties();
                 }
+
+                if (serviceType?.Type != null)
+                {
+                    MixedRealityDataProviderAttribute providerAttribute = MixedRealityDataProviderAttribute.Find(serviceType.Type) as MixedRealityDataProviderAttribute;
```

Comment:
Can the base `IsProfileRequired` be used here?

Reply:
likely.

that was the next change i was going to make 😄

Reply:
done. and found a bug in IsProfileRequired at the same time